sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -0,0 +1,214 @@
+# Azure Key Vault Certificate client library for .NET
+Azure Key Vault is a cloud service that provides secure storage and automated management of certificates used throughout a cloud application. Multiple certificate, and multiple versions of the same certificate, can be kept in the Key Vault. Each certificate in the vault has a policy associated with it which controls the issuance and lifetime of the certificate, along with actions to be taken as certificates near expiry.
+
+The Azure Key Vault Certificate client library enables programmatically managing certificates, offering methods to create, update, list and delete certificates, policies, issuers and contacts. The library also supports managing pending certificate operations, and management of deleted certificates.
```

Comment:
Nit: I recommend the Oxford comma everywhere for readability. It helps separate clauses, which is especially important with complex list statements such as this.